generators/client-2/templates/src/main/webapp/app/shared/_shared.ng2module.ts

```diff
@@ -24,12 +34,19 @@ import {
 @NgModule({
     imports: [
         <%=angular2AppName%>SharedLibsModule,
-        <%=angular2AppName%>SharedCommonModule,
+        <%=angular2AppName%>SharedCommonModule<%_ if (enableTranslation){ _%>,
+        TranslateModule.forRoot({
```

Comment:
This one has some wiring problems. Can you check this part. I don't know, moving this chunk over that file is causing problem.